eureka2-ext/eureka2-overrides-aws/src/test/java/com/netflix/eureka2/server/service/overrides/S3OutOfServiceOverridesSourceTest.java

```diff
@@ -0,0 +1,74 @@
+package com.netflix.eureka2.server.service.overrides;
+
+import com.netflix.eureka2.testkit.data.builder.SampleOverrides;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+
+import java.util.Map;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.*;
+
+/**
+ * Ignored as these tests needs an actual s3 backing to execute. Uncomment ignore and execute with the correct
```

Comment:
Given we have more tests like these, which are external systems integration tests, it would be good to define common runtime requirements, that once fulfilled would allow anybody to run those tests. In this particular case, AWS credentials would be sufficient.